src/FileService.Models.Db/DbFile.cs

```diff
@@ -33,6 +36,22 @@ public void Configure(EntityTypeBuilder<DbFile> builder)
             builder
                 .Property(p => p.Extension)
                 .IsRequired();
+
+            builder
+                .Property(p => p.CreatedAtUtc)
+                .IsRequired();
+
+            builder
```

Comment:
is not required. Required <=> not null. Also for structs you can don't write is required, because they cannot be null